src/protocol/codecs/binary.rs

```diff
@@ -75,13 +75,13 @@ pub struct BinaryMessage {
     pub payload: Payload,
```

Comment:
This filename should also be renamed from `binary.rs` to `message.rs`

Reply:
done